core/src/net/sf/openrocket/file/openrocket/savers/InnerTubeSaver.java

```diff
@@ -28,6 +28,7 @@ protected void addParams(net.sf.openrocket.rocketcomponent.RocketComponent c, Li
 
 		elements.add("<clusterconfiguration>" + tube.getClusterConfiguration().getXMLName()
 				+ "</clusterconfiguration>");
+		elements.add("<clusterseparation>" + tube.getClusterSeparation() + "</clusterseparation>");
 		elements.add("<clusterscale>" + tube.getClusterScale() + "</clusterscale>");
```

Comment:
Can you add a comment above this line: `// TODO: delete clusterscale when compatibility with OpenRocket 15.03 is not needed anymore`

Reply:
Added